azure-pipelines.yml

```diff
@@ -9,9 +9,11 @@ pool:
   vmImage: ubuntu-latest
 
 steps:
-- script: docker-compose build --build-arg PHP_VERSION=$PHP_VERSION ci && docker-compose run --rm ci
-  displayName: 'Run Tests PHP 8.2'
+- script: docker compose build dev
+  displayName: Build image
+
+- script: docker compose run --rm dev bash -c 'composer install && composer ci'
```

Comment:
tak tady by ale melo zustat ci

Reply:
nemelo, zadna `ci` service uz neexistuje (dev i ci prostredi jsou totozny)

Reply:
no ale nemají být, v ci nechceš mít namapovanej aktuální adresář

Reply:
Chces (muzes), tohle je libka, ne appka. Jak jsem psal, stejne to delame v platform-libraries.

Reply:
No jako nechceš, nic tam být nemá, tak se to nemá mapovat, nevím no kdy se tohle stalo. Hlavně, že tam máme profile ci a nic v něm :(